### PR TITLE
Create HowtoRAGAssistants.md

### PR DESCRIPTION
**Pull request by slinaresGX**
New article added.

### CODE REVIEW

File: saia-docs/docs/HowtoRAGAssistants.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+Next, click on the CONFIRM button.
+
+## Step 2: Set RAG Assistants
+
+Once the RAG Assistant is created, you return to the main page that provides the UPDATE option. This option allows you to customize the RAG 
+Assistant to your needs.
+
+![image](https://github.com/genexus-books/Saia/blob/fc9c844045233f79c3149780365a6100663c642b/saia-docs/assets/images/RAGAssistantsSection10.png?raw=true)
+
+For more information, see: RAG Assistants section
```
Comment (Valentina-Deambrosio10) [inline saia-docs/docs/HowtoRAGAssistants.md:37]: Shouldn't a link be placed here?

**Review by Valentina-Deambrosio10 — COMMENTED** [review]
One comment done in the review.

File: saia-docs/docs/HowtoRAGAssistants.md (hunk 1 above)
Comment (slinaresGX) [reply saia-docs/docs/HowtoRAGAssistants.md:37]: Yes, but I need the approval of RAG Assistants section (Re: [genexus-books/Saia] Create RAGAssistantsSection.md (PR #74)), in order to add link.

Comment (slinaresGX) [reply saia-docs/docs/HowtoRAGAssistants.md]: Done!

**Review by Valentina-Deambrosio10 — APPROVED** [review]
Approved.

**Pull request merged**